app/views/taxons/show_a.html.erb

```diff
@@ -1,5 +1,6 @@
 <% content_for :page_class, "taxon-page taxon-page--grid" %>
 <% content_for :is_taxon_with_subtopics, presented_taxon.show_subtopic_grid? %>
+<% content_for :brexit_taxon, presented_taxon.is_brexit? %>
```

Comment:
Does this need to be defined here? We could just query `presented_taxon.is_brexit?` directly? I think we've previously only defined these for things that need to be accessed outside the view (e.g: in the layout)